test/nntrainer_test_util.cpp

```diff
@@ -218,6 +218,30 @@ makeGraph(const std::vector<LayerRepresentation> &layer_reps) {
   return graph_rep;
 }
 
+nntrainer::GraphRepresentation
+makeGraph_V2(const std::vector<LayerRepresentation> &layer_reps) {
```

Comment:
How about rename it to explicitly let users knows what this would do? Such as makeCompledGraph?